### PR TITLE
Edit issues after intake

### PR DESCRIPTION
**Pull request by pkarman**
connects #11388 #11758 #11907 

### Description
Adds links to the async job page and user stats pages on every completed Intake and the Edit Issues page. This should help increase visibility as to who/when/what for each Intake.

Example edit page change:
Screen Shot 2019-09-04 at 9 00 56 AM

Example confirmation page change (the message about VBMS does not appear for Board Appeals):
Screen Shot 2019-09-07 at 7 12 47 AM

When the claim review has not finished processing, give a custom error message with a link to the job details page:
Screen Shot 2019-09-06 at 5 59 42 PM

### CODE REVIEW

**Comment by carodew** [comment]
qq on the screenshots -- 

What does the timestamp `Tuesday Sept. 3 18:04` link to? I assume it's a link because it's blue. 

Second, do we assume BVA users are used to using military time? (which may be a reasonable assumption given that I also assume that a fair number of them are Veterans).

**Comment by pkarman** [comment]
The timestamp link goes to the async job details page:

Screen Shot 2019-09-04 at 4 21 39 PM

re: 24-hour time, yes my bias is that it is the most concise and un-ambiguous, and that at VA folks are used to it. Happy to have that bias proved wrong. See thread in https://dsva.slack.com/archives/C754H751T/p1566998363000300

**Comment by carodew** [comment]
I have a bias towards non-military time for non-military U.S. humans' readability but since I have no evidence one way or another on what BVA's preference is, I see no reason to block this for that. Ship it!

File: app/controllers/asyncable_jobs_controller.rb [hunk 1]
```diff
@@ -72,6 +74,12 @@ def verify_access
     redirect_to "/unauthorized"
   end
 
+  def verify_job_access
+    return true if current_user.css_id == job&.asyncable_user
```
[the remark this answers is not in the document]
Comment (pkarman) [reply app/controllers/asyncable_jobs_controller.rb]: yes I only wanted to open this up in this PR to the affected user, beyond the existing access for Admin Intake. I'm happy to consider expanding it once we have concrete demand for it.

**Pull request merged**